lib/buildreq.c

```diff
@@ -175,7 +175,7 @@ int rc_aaa_ctx_server(rc_handle * rh, RC_AAA_CTX ** ctx, SERVER * aaaserver,
 
 		result = rc_send_server_ctx(rh, ctx, &data, msg, type);
 
-		if ((result == OK_RC) || (result == CHALLENGE_RC)) {
```

Comment:
Hi @nmav ,

Thanks for the response and support. I have added a testcase. Kindly review the changes and let me know your views